utils/maintenence.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
```

Comment:
So I think that perhaps the easy thing to do here is to just split up all of these tasks into their own shell scripts (eg maintainence-pyne.sh) and then have one shell script that runs them all (this one, maintainence.sh). Another alternative would be a simple makefile.